test/validation/api/classification/odp_classification_basic.c

```diff
@@ -733,6 +732,9 @@ static int check_capa_cos_hashing(void)
 	if (odp_cls_capability(&capa) != 0)
 		return ODP_TEST_INACTIVE;
 
+	if (capa.hash_protocols.all_bits == 0)
+		return ODP_TEST_INACTIVE;
```

Comment:
Checking the value of max_hash_queues is sufficient for testing if hash is supported. There should not be a case where max_hash_queues is greater than one but hash_protocols.all_bits is zero.